src/components/ec-timer/ec-timer.story.js

```diff
@@ -37,3 +37,21 @@ stopped.args = {
 stopped.parameters = {
   visualRegressionTests: { disable: true },
 };
+
+export const stoppedWithMinutes = Template.bind({});
+stoppedWithMinutes.args = {
+  seconds: 30,
+  showMinutes: true,
+  isRunning: false,
+};
+
+stoppedWithMinutes.parameters = {
+  visualRegressionTests: { disable: true },
+};
```

Comment:
can we have `visualRegressionTests` for this story as we have for the `basic`?

Reply:
@mcibique what is the logic to show/disable the visual regression tests for a specific story?

Reply:
Visual regression tests are very expensive to run in terms or generating pngs and comparing pngs, so we are trying to have them as small as possible and as less as possible.

If we can display all possible component states in one story (usually called `all`) then use that story for tests and ignore the rest, because it's duplicating what has been already tested (e.g. ec-input-field). If we don't have the `all` story, then we go with all other stories if they add any testing value (e.g. ec-timer).

I think the `ec-timer` doesn't have `all` story so we went with others but because `basic` and `stopped` stories renders the same png (animations are disabled while these tests are running so there's nothing more to test), we decided to save some time by ignoring the `stopped` story. If `stoppedWithMinutes` and `showMinutes` renders the same and we're sure that one of them covers all scenarios then we can keep only one of them.

Reply:
ok, thank you @mcibique